crates/sui-network/src/state_sync/mod.rs

```diff
@@ -128,6 +129,8 @@ struct PeerHeights {
     peers: HashMap<PeerId, PeerStateSyncInfo>,
     unprocessed_checkpoints: HashMap<CheckpointDigest, Checkpoint>,
     sequence_number_to_digest: HashMap<CheckpointSequenceNumber, CheckpointDigest>,
+
+    no_peer_to_sync_content_wait_internal: Duration,
```

Comment:
i got a bit confused reading this name, wondering if you can add comments to document what it's for or change the name to make it more obvious?

Reply:
this is a `Duration` to `sleep` when the node finds no peer can fulfill its content request. Will add comments